paasta_tools/mesos_tools.py

```diff
@@ -137,13 +137,13 @@ def filter_not_running_tasks(tasks):
     return [task for task in tasks if task['state'] != 'TASK_RUNNING']
 
 
-def get_running_tasks_from_active_frameworks(job_id):
+def get_running_tasks_from_active_frameworks(job_id=''):
     active_framework_tasks = get_current_tasks(job_id)
     running_tasks = filter_running_tasks(active_framework_tasks)
     return running_tasks
 
 
-def get_non_running_tasks_from_active_frameworks(job_id):
+def get_non_running_tasks_from_active_frameworks(job_id=''):
```

Comment:
@EvanKrall mesos cli gets jobs by checking if `job_id in task['id']`, so empty string matches all tasks.

It's pretty hacky and based off of `mesos-cli` (`mesos_tools.get_current_tasks` calls [this method](https://github.com/mesosphere/mesos-cli/blob/master/mesos/cli/master.py#L199)) which in the long term might become `dcos-cli` so idk about the longevity of this.